extensions/blocks/opentable/opentable.php

```diff
@@ -91,7 +91,21 @@ function add_language_setting() {
 function load_assets( $attributes ) {
 	\Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
 
-	$classes = \Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attributes );
+	$classes = array(
+		sprintf(
+			'wp-block-jetpack-%s-theme-%s',
+			FEATURE_NAME,
+			esc_attr( get_attribute( $attributes, 'style' ) )
+		),
+	);
+	if ( count( $attributes['rid'] ) > 1 ) {
+		$classes[] = 'is-multi';
+	}
+	$classes = \Jetpack_Gutenberg::block_classes(
+		FEATURE_NAME,
+		$attributes,
+		$classes
+	);
 	$content = '<div class="' . esc_attr( $classes ) . '">';
```

Comment:
The attribute actually gets escaped here. Shall I back out that last change @jeherve?

Reply:
Oh yeah, missed that one, sorry!

Reply:
No probs, I've removed the last commit